library/src/main/java/com/datatorrent/lib/db/AbstractTransactionableStoreOutputOperator.java

```diff
@@ -67,6 +68,8 @@ public void process(T t)
 
   };
 
+  public final transient DefaultOutputPort<T> error = new DefaultOutputPort<>();
```

Comment:
This was added to track the error tuples which fail to be processed  / inserted into database. It made sense to add it at the same place where the input port was declared.